test/verify/check-metrics

```diff
@@ -840,64 +840,64 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
 
         # No matching type
         m.execute("mkdir -p /tmp/hwmon/hwmon0")
-        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
+        self.addCleanup(m.execute, "rm -rf /tmp/hwmon")
+        m.write("/tmp/hwmon/hwmon0/name", "BAT0")
+        m.write("/tmp/hwmon/hwmon0/temp1_input", "40000")
         m.execute("mount -o bind /tmp /sys/class")
```

Comment:
This is a bit brittle -- there are often other files in /tmp which may disturb the layout. I recommend creating a new /tmp/sensor-sys-class/ or so directory for that, and only bind that, instead of the whole /sys. Sorry for missing that in the original PR.